### ISSUE TITLE
Not an Issue more of a question

### ISSUE THREAD

**Issue by wabj** [Flodu31/HomeAssistant-PeugeotIntegration#15]
Hi,

Thanks for this integration :)

Is there anyway to change km to mi at all?

**Comment by dreimer1986** [comment]
How about some calculation?
```
      e208_mileage:
        friendly_name: "Kilometerstand"
        unit_of_measurement: "km"
        value_template: "{{ ((state_attr('sensor.peugeot_e208', 'timed_odometer')['mileage']) / 1.609) | round(2)}}"
```

**Comment by Flodu31** [comment]
Thanks @dreimer1986 you have been quicker than me :D

**Comment by flobz** [comment]
@Flodu31 maybe you can add this tips to your doc?

**Comment by Flodu31** [comment]
Thanks. Done. Added it as comment

**Comment by wabj** [comment]
Thank you. 

I’m new to Home Assistant

Sent from my iPhone.

On 8 Nov 2021, at 06:33, Flodu31 ***@***.***> wrote:

﻿
Closed #15.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by dreimer1986** [comment]
> Thanks @dreimer1986 you have been quicker than me :D

You're welcome ^^ Just one small addition. Kilometerstand is German. I edited my code block to English now + mi as abbrevation.
